src/generate/generators/CoreFull.ts

```diff
@@ -39,9 +39,6 @@ export class AzCoreFullGenerator extends GeneratorBase {
                 files[
                     path.join(model.azOutputFolder, 'generated/_params.py')
                 ] = GenerateAzureCliParams(model, isDebugMode);
-                files[
-                    path.join(model.azOutputFolder, 'generated/commands.py')
-                ] = GenerateAzureCliCommands(model);
                 files[
                     path.join(model.azOutputFolder, 'generated/custom.py')
```

Comment:
Can we consistent about the name, e.g. AzOutputFolder

Reply:
sure

Reply:
Just found out this involves too many changes. Maybe we should hold on to change this until all the other files are changed into template based?